### ISSUE TITLE
Unable to connect to android.googleapis.com:443

### ISSUE THREAD

**Issue by colouredFunk** [davibennun/laravel-push-notification#50]
Everything is working fine on my localhost but when uploaded to my server I'm getting - 

Unable to connect to android.googleapis.com:443 . Error #0: stream_socket_client(): unable to connect to android.googleapis.com:443 (A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. )

**Comment by anil1712** [comment]
Yes, I am also getting the same issue? Did you got any solution for the same. Please help me to out of this problem. 

Thanks

**Comment by colouredFunk** [comment]
I've got the same issue a year later!! Not sure what happened? Anyone getting this?
